zamba/models/cnnensemble/src/prepare_train_data.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import StratifiedKFold
+import random
+import os
+from PIL import Image
+from multiprocessing import Pool
+from tqdm import tqdm
+
+from zamba.models.cnnensemble.src import config
+from zamba.models.cnnensemble.src import utils
+
+
+def generate_folds(config):
+    training_set_labels_ds_full = pd.read_csv(config.TRAINING_SET_LABELS)
+    training_set_labels_ds_full['fold'] = 0
+
+    data = training_set_labels_ds_full.as_matrix(columns=config.CLASSES + ['filename', 'fold'])
+
+    cls = np.argmax(data[:, :-2], axis=1)
```

Comment:
Change variable name so this doesn't conflict with the conventional `cls` for a `Class` in `classmethod`.